Release notes for 8.9.0 (#15162)
elastic/logstash: docs/static/releasenotes.asciidoc

* <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>

[[logstash-8-9-0]]
=== Logstash 8.9.0 Release Notes

[[notable-8.9.0]]
==== Notable issues fixed

* Fix an issue where installs and updates of certain logstash plugins may fail behind a proxy https://github.com/elastic/logstash/pull/15131[#15131]. This issue was manifesting after a recent change to `logstash-filter-translate` updated to require the use of the `jar-dependencies` gem to retrieve artifacts from maven when installing the plugin, and could prevent the update of this plugin when a proxy is in use.
* Improve logging when logstash is stalled on shutdown https://github.com/elastic/logstash/pull/15056[#15056]. This commit adds information about the main thread, if it is that one that is causing the shutdown to be stalled.

> * Improve SSL settings for connection to elasticsearch for central management and monitoring https://github.com/elastic/logstash/pull/15045[#15045]. This commit adds settings support for file-based certificates and cipher suites for management and monitoring settings, and removes the deprecation warnings from the logs that have been since SSL configuration settings were revamped in the Elasticsearch output.

Reviewer: ```suggestion
* Improved SSL settings for connection to {es} for central management and monitoring https://github.com/elastic/logstash/pull/15045[#15045]. 
This commit adds settings support for file-based certificates and cipher suites for management and monitoring settings, and removes the deprecation warnings from the logs that have been in since SSL configuration settings were revamped in the {es} output.
```

[[dependencies-8.9.0]]
==== Updates to dependencies

* Update Bundler to version 2.4 https://github.com/elastic/logstash/pull/14995[#14995]
* Update snakeyaml to `2.0`
* Update commons-io to `2.13.0`
* Update commons-compress to `1.23.0`
* Update jrjackson to `0.4.18`
* Update jackson to `2.15.2`
* Update databind to `2.15.2`

==== Plugins

*Azure_event_hubs Input - 1.4.5*

* Update multiple dependencies such as `gson`, `log4j2`, `jackson` https://github.com/logstash-plugins/logstash-input-azure_event_hubs/pull/83[#83]

*Beats Input - 6.6.3*

> * Docs: Updated the `ssl_client_authentication` and `ssl_verify_mode` documentation explaining that CN and SAN are not validated. https://github.com/logstash-plugins/logstash-input-beats/pull/473[#473]

Reviewer: ```suggestion
* [DOC] Updated the `ssl_client_authentication` and `ssl_verify_mode` documentation explaining that CN and SAN are not validated. https://github.com/logstash-plugins/logstash-input-beats/pull/473[#473]
```

Reviewer: This is the current format we use in changelogs and for plugin releases.

* Update netty to 4.1.94 and jackson to 2.15.2 https://github.com/logstash-plugins/logstash-input-beats/pull/474[#474]

*Http Input - 3.7.2*

* Update netty to 4.1.94 https://github.com/logstash-plugins/logstash-input-http/pull/167[#167]

*Snmp Input - 1.3.2*

> * Docs: Add troubleshooting help for "failed to locate MIB module" error when using smidump to convert MIBs

Reviewer: ```suggestion
* [DOC] Add troubleshooting help for "failed to locate MIB module" error when using smidump to convert MIBs https://github.com/logstash-plugins/logstash-input-snmp/pull/112[#112]
```

[[notable-8.9.0]]
==== Notable issues fixed

> * Fixed an [issue ](https://github.com/elastic/logstash/issues/15130) where installs and updates of certain {ls} plugins could fail when located behind a proxy https://github.com/elastic/logstash/pull/15131[#15131]. 

Reviewer: ```suggestion
* Fixed an https://github.com/elastic/logstash/issues/15130[issue] where installs and updates of certain {ls} plugins could fail when located behind a proxy https://github.com/elastic/logstash/pull/15131[#15131]. 
```

> * Fixed an [issue ](https://github.com/elastic/logstash/issues/15130) where installs and updates of certain {ls} plugins could fail when located behind a proxy https://github.com/elastic/logstash/pull/15131[#15131]. 

Reviewer: ```suggestion
* Fixed an https://github.com/elastic/logstash/issues/15130[issue]  where installs and updates of certain {ls} plugins could fail when located behind a proxy https://github.com/elastic/logstash/pull/15131[#15131]. 
```
